src/Service.GraphQLBuilder/Sql/SchemaConverter.cs

```diff
@@ -220,7 +220,7 @@ public static string GetGraphQLTypeFromSystemType(Type type)
             return type.Name switch
             {
                 "String" => STRING_TYPE,
-                "Guid" => STRING_TYPE,
+                "Guid" => ID_TYPE,
```

Comment:
Why `ID_TYPE` and not `UUID_TYPE` https://chillicream.com/docs/hotchocolate/v12/defining-a-schema/scalars/#uuid-type

Reply:
Seemed to be the appropriate choice. Thanks @Aniruddh25 for making the changes.